v2/cmd/functional-test/testcases.txt

```diff
@@ -0,0 +1,24 @@
+127.0.0.1 {{binary}} -silent -p 8000
+127.0.0.1 {{binary}} -silent -il
+127.0.0.1 {{binary}} -silent -l test-data/request.txt -p 8000
+127.0.0.1 {{binary}} -silent -p 8000,443
+127.0.0.1 {{binary}} -silent -tp 100 -p 8000
+127.0.0.1 {{binary}} -silent -ep 200  -p 8000
+127.0.0.1 {{binary}} -silent -ec 100,400  -p 8000
+127.0.0.1 {{binary}} -silent -version
+127.0.0.1 {{binary}} -silent -retries 5  -p 8000
+127.0.0.1 {{binary}} -silent -c 25  -p 8000
+127.0.0.1 {{binary}} -silent -verify  -p 8000
+127.0.0.1 {{binary}} -silent -timeout 30  -p 8000
+127.0.0.1 {{binary}} -silent -r test-data/request.txt  -p 8000
+127.0.0.1 {{binary}} -silent -warm-up-time 2  -p 8000
+127.0.0.1 {{binary}} -silent -nmap-cli 'nmap -sV'  -p 8000
```

Comment:
Please remove the double space between CLI args

Reply:
Done

Reply:
Done